Made PS to download createazureautomationaccount.ps1 more readable

michawets opened the pull request:
I made the PS to download the createazureautomationaccount.ps1 in section "Create an Azure Automation account" part 3 a little more readable for the users.

michawets committed:
Made PS to download createazureautomationaccount.ps1 more readable

I made the PS to download the createazureautomationaccount.ps1 in section "Create an Azure Automation account" part 3 a little more readable for the users.

ChristianMontoya commented on articles/virtual-desktop/set-up-scaling-script.md:
```diff
@@ -78,7 +78,9 @@ First, you'll need an Azure Automation account to run the PowerShell runbook. He
-     Invoke-WebRequest -Uri "https://raw.githubusercontent.com/Azure/RDS-Templates/master/wvd-templates/wvd-scaling-script/createazureautomationaccount.ps1" -OutFile "your local machine path\ createazureautomationaccount.ps1"
+     Set-Location -Path "c:\temp"
+$uri = "https://raw.githubusercontent.com/Azure/RDS-Templates/master/wvd-templates/wvd-scaling-script/createazureautomationaccount.ps1"
+Invoke-WebRequest -Uri $uri -OutFile ".\createazureautomationaccount.ps1"
```
@michawets : Can you indent these so that they all appear together in the same code block?

michawets replied:
Hi @ChristianMontoya , I totally forgot the indentation 😊 Should be fixed now

ChristianMontoya reviewed (changes requested):
Like the suggestion! However, please indent so the code block shows up together.

michawets committed:
Made PS to download createazureautomationaccount.ps1 more readable & Fixed the indentation

Made PS to download createazureautomationaccount.ps1 more readable & Fixed the indentation

ChristianMontoya reviewed (approved):
Looks good now!
